### ISSUE TITLE
Excuse me, are you sure your demo can load videos? I think the code in the demo is for loading image images

### ISSUE THREAD

**Issue by yanyide123** [jiawen-zhu/HQTrack#15]

**Comment by Gazuz33** [comment]
> _No description provided._

You are right, you need to add a small block of code that converts the video into a sequential set of pictures, and then combine the processed pictures into a video. The demo file works pretty well, but it's very slow on my RTX3060

**Comment by yanyide123** [comment]
> > _No description provided._
> 
> You are right, you need to add a small block of code that converts the video into a sequential set of pictures, and then combine the processed pictures into a video. The demo file works pretty well, but it's very slow on my RTX3060
There is another question, when using continuous images, is there a confirmation shortcut key to continue the program after the mouse hovers over the image annotation box or after clicking.
（还有一个问题，在使用连续的图像时候，运行开始后，鼠标在图像标注框或者点以后，是否有确认快捷键让程序继续运行。）

**Comment by Gazuz33** [comment]
> 
this is a part of demo code: 
    cv2.setMouseCallback('demo', OnMouse)
            k = cv2.waitKey(1)
            if k == ord('r'):
                break
that's why u need to press 'r' button to continue the program after box or point clicking

**Comment by Gazuz33** [comment]
> > > _No description provided._
> > 
> > 
> > You are right, you need to add a small block of code that converts the video into a sequential set of pictures, and then combine the processed pictures into a video. The demo file works pretty well, but it's very slow on my RTX3060
> > There is another question, when using continuous images, is there a confirmation shortcut key to continue the program after the mouse hovers over the image annotation box or after clicking.
> > （还有一个问题，在使用连续的图像时候，运行开始后，鼠标在图像标注框或者点以后，是否有确认快捷键让程序继续运行。）

this is a part of demo code:
cv2.setMouseCallback('demo', OnMouse)
k = cv2.waitKey(1)
if k == ord('r'):
break
that's why u need to press 'r' button to continue the program after box or point clicking

**Comment by yanyide123** [comment]
> 

Okay, that's it. Thank you

**Comment by qgq99** [comment]
I wonder if this demo script can run on Linux os? By the way, HQTrack is a great job.

**Comment by yanyide123** [comment]
> I wonder if this demo script can run on Linux os? By the way, HQTrack is a great job.

It can run on a Linux system

**Comment by qgq99** [comment]
> > I wonder if this demo script can run on Linux os? By the way, HQTrack is a great job.
> 
> It can run on a Linux system

Excuse me, I mean, it seems that it can't run on a Linux OS without GUI, or I make some mistake?

**Comment by yanyide123** [comment]
> > > I wonder if this demo script can run on Linux os? By the way, HQTrack is a great job.
> > 
> > 
> > It can run on a Linux system
> 
> Excuse me, I mean, it seems that it can't run on a Linux OS without GUI, or I make some mistake?

You can save the output file to a file
